admin-ui/src/component/ReservationUnits/ReservationUnitEditor/SortedSelect.tsx

```diff
@@ -12,7 +12,11 @@ function SortedSelect<T>(
     const opts = [...originalOptions];
     if (props.sort) {
       opts.sort((a, b) =>
-        a.label.toLowerCase().localeCompare(b.label.toLowerCase())
+        a.label && b.label
```

Comment:
Maybe the sorting logic could be moved to a helper function to minimize nested ternarys in code - applies also to the one in [admin-ui/src/component/my-units/UnitCalendar.tsx]